README.md

```diff
@@ -98,10 +98,9 @@ These widgets can be customized, added, removed and even reordered
 
 ### Arch
 
-The package is available on the AUR:
-
-- [swaync](https://aur.archlinux.org/packages/swaync/)
-- [swaync-git](https://aur.archlinux.org/packages/swaync-git/)
```

Comment:
We should leave the swaync-git information up for people who like to live on the bleeding edge :)

Reply:
Done [here](https://github.com/ErikReider/SwayNotificationCenter/commit/f67e704082e2498ea026fe7b738f6df4af334105) :)